src/core/Link/BaseLink/BaseLink.baseStyles.ts

```diff
@@ -1,29 +1,31 @@
 import { css } from 'styled-components';
 import { SuomifiTheme } from '../../theme';
-import { element, font } from '../../theme/reset';
```

Comment:
The `element` function seems to only add `color: ${theme.colors.blackBase};`? I think it could be removed in this case

Reply:
I think so too. I didn't see any adverse effects from it, since the color seems to be defined by other means in all cases.